src/screens/Home/Home.js

```diff
@@ -1,788 +1,104 @@
 // @flow
 /*
     Pillar Wallet: the personal data locker
-    Copyright (C) 2019 Stiftung Pillar Project
+    Copyright (C) 2021 Stiftung Pillar Project
```

Comment:
Are you sure? Sometimes copyright date must match the registration, but not publish date, do you think we should check this with product team? cc @IAmKio